### PR TITLE
added treemaker 1.0

### PR DESCRIPTION
**Pull request by AnnaSyme**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/treemaker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    - treemaker --help
+
+  # You can also put a file called run_test.py in the recipe that will be run
+  # at test time.
+
+
+about:
+  home: https://github.com/SimonGreenhill/treemaker
+  license: BSD License
```
Comment (bgruening) [inline recipes/treemaker/meta.yaml]: `License` should not be part of the name.

File: recipes/treemaker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  run:
+    - python
+
+test:
+  # Python imports
+  imports:
+    - treemaker
+
+  commands:
+    # You can put test commands to be run here.  Use this to test that the
```
Comment (bgruening) [inline recipes/treemaker/meta.yaml]: Can you please cleanup the recipe by removing the comments.

File: recipes/treemaker/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (bgruening) [inline recipes/treemaker/build.sh]: Please see this section: https://bioconda.github.io/linting.html#setup-py-install-args
and change this to `$PYTHON setup.py install --single-version-externally-managed --record=record.txt`

File: recipes/treemaker/build.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
+
+# Add more build steps here, if they are necessary.
+
+# See
```
Comment (bgruening) [inline recipes/treemaker/build.sh]: Can you please cleanup this as well?

File: recipes/pyseer/build.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```
Comment (bgruening) [inline recipes/pyseer/build.sh]: https://bioconda.github.io/linting.html#setup-py-install-args :) see below

File: recipes/pyseer/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    - pyseer
+
+  commands:
+
+    - pyseer --help
+    - square_mash --help
+
+about:
+  home: https://github.com/mgalardini/pyseer
+  license: GNU General Public License v3 (GPLv3)
```
Comment (bgruening) [inline recipes/pyseer/meta.yaml]: GPLv3 is ok.

File: recipes/pyseer/bld.bat [hunk 7]
```diff
@@ -0,0 +1,8 @@
+"%PYTHON%" setup.py install
```
Comment (bgruening) [inline recipes/pyseer/bld.bat]: This file can be removed as we do not build for Windows on BioConda.

**Comment by johanneskoester** [comment]
We have made changes to our build system. Please update your fork with the latest upstream/master.

**Comment by druvus** [comment]
@AnnaSyme a newer version of treemaker was added in #7488 so I am closing this.

**Pull request closed without merging**